src/remapping.ts

```diff
@@ -36,9 +36,10 @@ export default function remapping(
   input: SourceMapInput | SourceMapInput[],
   loader: SourceMapLoader,
   excludeContent?: boolean,
-  decodeMappings?: boolean
+  decodeMappings?: boolean,
+  segmentsAreSorted?: boolean
```

Comment:
If we're going to do this, we should probably create an options object to hold all the flags.